cpp/src/arrow/c/bridge.h

```diff
@@ -218,6 +218,9 @@ Status ExportDeviceRecordBatch(const RecordBatch& batch,
 using DeviceMemoryMapper =
     std::function<Result<std::shared_ptr<MemoryManager>>(ArrowDeviceType, int64_t)>;
 
+Result<std::shared_ptr<MemoryManager>> DefaultDeviceMapper(ArrowDeviceType device_type,
```

Comment:
Should probably add `ARROW_EXPORT` to fix the Windows issue.

Reply:
Thanks, that worked!